tests/config/ray-cluster.ray-ft.yaml.template

```diff
@@ -145,3 +162,115 @@ spec:
                 periodSeconds: 3
                 timeoutSeconds: 1
                 failureThreshold: 300
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-script
+data:
+  test_ray_serve_1.py: |
+    import requests
+    from starlette.requests import Request
+    import ray
+    from ray import serve
+    import time
+    import sys
+
+    # 1: Define a Ray Serve model.
+    @serve.deployment(route_prefix="/")
+    class MyModelDeployment:
+        def __init__(self, msg: str):
+            self._msg = msg
+
+        def __call__(self):
+            return self._msg
+
+    ray.init(address='ray://127.0.0.1:10001', namespace=sys.argv[1])
```

Comment:
Since we're connecting from within the head pod, we should probably use the GCS server port, 6379, rather than the Ray client port 10001.

Reply:
Probably need to remove `ray://` too, otherwise it will try to connect using Ray Client (which we're trying to deemphasize in Ray)

Reply:
Updated to `ray.init(namespace=sys.argv[1])`. `ray.init()` will choose port 6379 by default.

```sh
➜  kuberay git:(move-script-to-configmap) ✗ kubectl exec -it raycluster-external-redis-head-s4bwx -- bash
(base) ray@raycluster-external-redis-head-s4bwx:~$ echo $RAY_ADDRESS
127.0.0.1:6379
```